algobot/helpers.py

```diff
@@ -40,6 +44,49 @@
 LONG_INTERVAL_MAP = {v: k for k, v in SHORT_INTERVAL_MAP.items()}
 
 
+class Paths:
+
+    def __init__(self, root_dir: str, app_dirs):
+        self.root_dir = root_dir
+        self.app_dirs = app_dirs
+
+    def get_ui_dir(self) -> str:
+        return os.path.join(self.root_dir, 'UI')
+
+    def get_log_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_log_dir, 'Logs')
+
+    def get_database_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'Databases')
+
+    def get_state_path(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'state.json')
+
+    def get_optimizer_results_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'Optimizer Results')
+
+    def get_backtest_results_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'Backtest Results')
+
+    def get_trade_history_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'Trade History')
+
+    def get_volatility_results_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'Volatility Results')
+
+    def get_csv_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'CSV')
+
+    def get_configuration_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'configuration')
+
+    def get_credentials_dir(self) -> str:
+        return os.path.join(self.app_dirs.user_data_dir, 'Credentials')
+
+
+PATHS = Paths(ROOT_DIR, AppDirs(APP_NAME, APP_AUTHOR))
```

Comment:
We could prob mock out this for testing to avoid interactions with stuff in the user home dir

Reply:
Yep, that makes sense

Reply:
Done